spark/pom.xml

```diff
@@ -786,6 +781,68 @@
         </executions>
       </plugin>
 
+      <!-- for pyspark -->
+      <plugin>
+        <groupId>com.googlecode.maven-download-plugin</groupId>
+        <artifactId>download-maven-plugin</artifactId>
+        <version>1.2.1</version>
+        <executions>
+          <execution>
+            <id>download-pyspark-files</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+            <configuration>
+              <url>${spark.download.url}</url>
+              <unpack>true</unpack>
+              <outputDirectory>${project.build.directory}/spark-dist</outputDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
download-maven-plugin uses cache so we can use this several times without downloading. In case of ant task, it doesn't support caching.